schema.proto

```diff
@@ -99,13 +100,35 @@ message TermV2 {
     bool bool = 6;
     TermSet set = 7;
     Empty null = 8;
+    Array array = 9;
```

Comment:
this should already be part of the `dev` branch, maybe you have started your work on an outdated version?

Reply:
yup, this was fixed by merging dev in here